### ISSUE TITLE
Do you need maintainers?

### ISSUE THREAD

**Issue by wub** [casesandberg/react-color#928]
Hey @casesandberg, thanks for the great library. I've noticed that there's 38 PRs and a heap of issues. Are you looking for help maintaining this library?

I note that this has been asked 3 years ago as well: https://github.com/casesandberg/react-color/issues/823

Cheers

**Comment by Aichnerc** [comment]
Yeah @casesandberg would be great for this package to get at least some basic maintenance to keep it working in future React versions.
